ios/RNPrint/RNPrint.m

```diff
@@ -100,29 +137,35 @@ -(void)launchPrint:(NSData *) data
     } else {
         _jobName = @"Document";
     }
+
+    if (options[@"baseUrl"]) {
+        _baseUrl = [NSURL URLWithString:options[@"baseUrl"]];
+    }
+    
+    if (options[@"useWebView"]) {
+        _useWebView = [[RCTConvert NSNumber:options[@"useWebView"]] boolValue];
```

Comment:
Do we need to have a `useWebView` prop at all? Can we just check to see if `baseUrl` is provided and handle it accordingly? e.g

```
if (_baseUrl) {
        _webView = [[WKWebView alloc] initWithFrame:self.bounds];
        _webView.navigationDelegate = self;
        [self addSubview:_webView];
        dispatch_async(dispatch_get_main_queue(), ^{
                [_webView loadHTMLString:_htmlString baseURL:_baseUrl];
        });
```

Reply:
Resolved https://github.com/christopherdro/react-native-print/pull/168/commits/ab35d20128937ba4d2ce39920fe68b993ebd57e7